warden_daemon/src/main.rs

```diff
@@ -43,7 +43,7 @@ struct Cli {
     realm_connection_wait_time_secs: u64,
 }
 
-#[tokio::main]
+#[tokio::main(flavor = "multi_thread")]
```

Comment:
According to the documentation at https://docs.rs/tokio/latest/tokio/attr.main.html#multi-threaded-runtime
The "multi_thread" flavor is the default one. Is there any reason why you enable it explicitly?

Reply:
I just wanted to make it explicit to signal someone who will be working on this project in the future that this option is mandatory - warden daemon can crash if this option is set to `#[tokio::main(flavor = "current_thread")]` due to the latest changes and adding `block_in_place`.
If you think that it's not necessary I can remove it.